src/main/java/storage/ExcludeInSerializationAnnotation.java

```diff
@@ -0,0 +1,11 @@
+package storage;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Annotation to be used to indicate that a particular class attribute should not be serialized by <code>Gson</code>
+ */
+@Retention(RetentionPolicy.RUNTIME) // makes the annotation available at runtime
+public @interface ExcludeInSerializationAnnotation {
```

Comment:
Interesting syntax! Seems like it is just a plain decorator for identification and exclusion in serialization?

Reply:
Not sure what a plain decorator is, but yea I think the idea is that this annotation serves as a sort of flag to let Gson know what should not be serialized.